cinema_service/urls.py

```diff
@@ -1,8 +1,11 @@
 from django.contrib import admin
 from django.urls import path, include
+from rest_framework.authtoken import views
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("api/token-auth/", views.obtain_auth_token, name="token-auth"),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
```

Comment:
The `include` function for 'cinema.urls' specifies a namespace, but ensure that the 'cinema/urls.py' file defines an `app_name` variable to match this namespace. This helps prevent URL name conflicts across different apps.